config/class-sync.php

```diff
@@ -65,6 +65,7 @@ public function filter_cron_schedules( $schedule ) {
 
 	public function do_cron() {
 		$this->maybe_sync_jetpack_privacy_settings();
+		$this->put_site_details();
```

Comment:
Should we restrict this to a one (or a few sites) to start to make sure everything's working correctly?